testing/TestHarness/TestHarness.Windows/TestHarness.Windows.csproj

```diff
@@ -41,6 +41,7 @@
 		<PackageReference Include="IdentityModel.OidcClient" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Console" />
 		<PackageReference Include="Microsoft.Identity.Client" />
+		<PackageReference Include="Microsoft.Web.WebView2"/>
```

Comment:
if this reference is needed we should be adding the dependency to the Auth package

Reply:
Not required, just added to see if I could force a version update to see if there was a fix for build error